redfish-core/lib/log_services.hpp

```diff
@@ -1880,31 +1906,72 @@ inline void requestRoutesDBusCELogEntryCollection(App& app)
         });
 }
 
-inline void
-    updateResolvedProperty(const crow::Request& req,
+inline void updateProperty(const crow::Request& req,
                            const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
                            const std::string& entryId)
 {
     std::optional<bool> resolved;
-    if (!json_util::readJson(req, asyncResp->res, "Resolved", resolved))
+    std::optional<nlohmann::json> oemObject;
+#ifdef BMCWEB_ENABLE_IBM_MANAGEMENT_CONSOLE
+    std::optional<bool> managementSystemAck;
+#endif
+    if (!json_util::readJson(req, asyncResp->res, "Resolved", resolved, "Oem",
+                             oemObject))
     {
         return;
     }
-    BMCWEB_LOG_DEBUG << "Set Resolved";
-    crow::connections::systemBus->async_method_call(
-        [asyncResp](const boost::system::error_code ec) {
-            if (ec)
-            {
-                BMCWEB_LOG_DEBUG << "DBUS response error " << ec;
-                messages::internalError(asyncResp->res);
-                return;
-            }
-        },
-        "xyz.openbmc_project.Logging",
-        "/xyz/openbmc_project/logging/entry/" + entryId,
-        "org.freedesktop.DBus.Properties", "Set",
-        "xyz.openbmc_project.Logging.Entry", "Resolved",
-        std::variant<bool>(*resolved));
+    if (resolved.has_value())
+    {
+        BMCWEB_LOG_DEBUG << "Set Resolved";
+        crow::connections::systemBus->async_method_call(
+            [asyncResp, entryId](const boost::system::error_code ec) {
+                if (ec)
+                {
+                    BMCWEB_LOG_DEBUG << "DBUS response error " << ec;
+                    messages::internalError(asyncResp->res);
+                    return;
+                }
+            },
+            "xyz.openbmc_project.Logging",
+            "/xyz/openbmc_project/logging/entry/" + entryId,
+            "org.freedesktop.DBus.Properties", "Set",
+            "xyz.openbmc_project.Logging.Entry", "Resolved",
+            std::variant<bool>(*resolved));
+    }
+#ifdef BMCWEB_ENABLE_IBM_MANAGEMENT_CONSOLE
+    if (oemObject)
+    {
+        std::optional<nlohmann::json> bmcOem;
+        if (!json_util::readJson(*oemObject, asyncResp->res, "OpenBMC", bmcOem))
+        {
+            return;
+        }
+        if (!json_util::readJson(*bmcOem, asyncResp->res, "ManagementSystemAck",
+                                 managementSystemAck))
+        {
+            BMCWEB_LOG_ERROR << "Could not read managementSystemAck";
+            return;
+        }
+    }
+    if (managementSystemAck.has_value())
+    {
+        BMCWEB_LOG_DEBUG << "Set ManagementSystemAck";
+        crow::connections::systemBus->async_method_call(
+            [asyncResp, entryId](const boost::system::error_code ec) {
```

Comment:
`entryId` does not need to pass inside captures of lambda

Reply:
Done